### PR TITLE
ci: add processor HA test

### LINKED ISSUE #407: Ensure the tables are rebalanced when a processor is stopped
# Cases

- [x] Stop a processor by admin-job

# Expectation
* The tables on the stopped processor are rebalanced

### PR DESCRIPTION
**Pull request by shafreeck**
This is a PR based on #428

### What problem does this PR solve? 

Add test for stopping and resuming the processors

Closes #407

### CODE REVIEW

File: tests/availability/processor.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    echo "changefeed id:" $changefeed
+
+    # stop the change feed job
+    # use ensure to wait for the change feed loading into memory from etcd
+    ensure $MAX_RETRIES "curl -s -d \"cf-id=$changefeed&admin-job=1\" http://127.0.0.1:8300/capture/owner/admin | grep true"
+
+    run_sql "INSERT INTO test.availability(id, val) VALUES (4, 4);"
+
+    # resume the change feed job
+    curl -d "cf-id=$changefeed&admin-job=2" http://127.0.0.1:8300/capture/owner/admin
```
Comment (amyangfei) [inline tests/availability/processor.sh:41]: should we wait for the server is recovered and it has campaigned to be owner again. I found the test case fails sometimes

Comment (shafreeck) [reply tests/availability/processor.sh:41]: The ownership was not changed ever, stopping a change feed does not resign an owner. Can you reproduce the fail case?

Comment (amyangfei) [reply tests/availability/processor.sh:41]: It happens often, only find and no any processor is created.
```
[2020/04/07 14:50:33.589 +08:00] [INFO] [owner.go:365] ["handle admin job"] [changefeed=373f0adf-0bcf-421f-92d0-5c39fd0208a7] [type="stop changefeed"]
[2020/04/07 14:50:33.591 +08:00] [INFO] [etcd.go:200] ["show check lock"] [status="{\"table-infos\":[{\"id\":45,\"start-ts\":415823882176430095}],\"table-p-lock\":null,\"table-c-lock\":null,\"admin-job-type\":0}"]
```
But if sleep 1s before `resume job`, it always successes

Comment (amyangfei) [reply tests/availability/processor.sh:41]: It happens often, only find and no any processor is created.
```
[2020/04/07 14:50:33.589 +08:00] [INFO] [owner.go:365] ["handle admin job"] [changefeed=373f0adf-0bcf-421f-92d0-5c39fd0208a7] [type="stop changefeed"]
[2020/04/07 14:50:33.591 +08:00] [INFO] [etcd.go:200] ["show check lock"] [status="{\"table-infos\":[{\"id\":45,\"start-ts\":415823882176430095}],\"table-p-lock\":null,\"table-c-lock\":null,\"admin-job-type\":0}"]
```
But if sleep 1s before `resume job`, it always successes

**Comment by shafreeck** [comment]
#442 fails the integration test

**Pull request merged**